store/postgres/job_spec_repository.go

```diff
@@ -63,6 +63,27 @@ func (repo *ProjectJobSpecRepository) GetByName(ctx context.Context, name string
 	return jobSpec, namespaceSpec, nil
 }
 
+func (repo *ProjectJobSpecRepository) GetByIDs(ctx context.Context, jobIDs []uuid.UUID) ([]models.JobSpec, error) {
+	var jobs []Job
+	if err := repo.db.WithContext(ctx).Where("project_id = ? AND job.id in ?", jobIDs).First(&jobs).Error; err != nil {
```

Comment:
Are we able to query multiple jobs using `First` ?

As per gorm docs:
GORM provides First, Take, Last methods to retrieve a single object from the database, it adds LIMIT 1 condition when querying the database

Reply:
you are right, instead of `First` this should use `Find`. updated.